docs/source/advanced_config.rst

```diff
@@ -31,6 +31,66 @@ We do this using the `source` argument, which most Fab steps accept.
     ])
 
 
+Parser Workarounds
+==================
+
+Unrecognised Dependencies
+-------------------------
+If a language parser is not able to recognise a dependency within a file,
+then Fab won't know the dependency needs to be compiled.
+For example, some versions of fparser don't recognise a call on a one-line if statement.
+In this case we can manually add the dependency using the `unreferenced_deps` argument to
+:class:`~fab.steps.analyse.Analyse`.
+
+Pass in the name of the called function.
+Fab will find the file containing this symbol and add it to the build.
+
+.. code-block::
+    :linenos:
+    :emphasize-lines: 3
+
+    config.steps = [
+        ...
+        Analyse(root_symbol='my_prog', unreferenced_deps=['my_func'])
+        ...
+    ]
+
+Unparsable Files
+----------------
+If a language parser is not able to process a file at all,
+then Fab won't know about any of its symbols and dependencies.
+This can sometimes happen to files which *compilers are* able to process.
+In this case we can manually give Fab the analysis results it should have got from the parser
+using the `special_measure_analysis_results` argument to :class:`~fab.steps.analyse.Analyse`.
+
+Pass in a :class:`~fab.dep_tree.ParserWorkaround` object containing the symbol definitions and dependencies of the file.
```

Comment:
The code shows a list of `ParserWorkaround` objects but this text implies that only a single object may be passed.